packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,19 +49,50 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
-    @attr({
-        attribute: 'direction'
-    })
+    @attr()
+    public override lang = '';
+
+    @attr()
     public direction: Direction = Direction.ltr;
```

Comment:
same as above, we should not be writing to the attribute because then it breaks normal inheritance